exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetPushDownFilter.java

```diff
@@ -165,13 +166,26 @@ protected void doOnMatch(RelOptRuleCall call, FilterPrel filter, ProjectPrel pro
       return;
     }
 
-
-    RelNode newScan = ScanPrel.create(scan, scan.getTraitSet(), newGroupScan, scan.getRowType());;
+    RelNode newScan = ScanPrel.create(scan, scan.getTraitSet(), newGroupScan, scan.getRowType());
 
     if (project != null) {
       newScan = project.copy(project.getTraitSet(), ImmutableList.of(newScan));
     }
-    final RelNode newFilter = filter.copy(filter.getTraitSet(), ImmutableList.<RelNode>of(newScan));
-    call.transformTo(newFilter);
+
+    List<RowGroupInfo> rowGroupInfos = ((AbstractParquetGroupScan)newGroupScan).rowGroupInfos;
```

Comment:
1. Please add space after `(AbstractParquetGroupScan)`.
2. Please try to get rid of explicit casting.

Reply:
Done